elevate/compat.py

```diff
@@ -0,0 +1,5 @@
+from django.utils.http import is_safe_url
+
+def is_safe_url(url, allowed_hosts, require_https):
+    # This is the compat function for older Django versions
+    return is_safe_url(url=url, host=allowed_hosts)
```

Comment:
You're redefining `is_safe_url` with this function, try using `from django.utils import http` and then calling `http.is_safe_url` to avoid this.